ScoutSuite/providers/aws/facade/cloudformation.py

```diff
@@ -24,6 +24,12 @@ async def get_stacks(self, region: str):
         finally:
             return stacks
 
+    async def get_stacksets(self, region: str):
+        stacksets = await AWSFacadeUtils.get_all_pages(
+            'cloudformation', region, self.session, 'list_stack_sets', 'Summaries')
+      
+        return stacksets
+
```

Comment:
Has this been tested at all?

Reply:
Yeah, I made some resources to test this.